### PR TITLE
ACS Chat SDK Readme

### PR DESCRIPTION
**Pull request by anuchandy**

### CODE REVIEW

**Review by JianpingChen — APPROVED** [review]

File: sdk/communication/azure-communication-chat/README.md [hunk 1]
```diff
@@ -0,0 +1,712 @@
+# Azure Communication Chat Service client library for Android
+This package contains the Android SDK for Azure Communication Services for Chat.
+Read more about Azure Communication Services [here](https://docs.microsoft.com/azure/communication-services/overview).
+
+# Getting started
+
+## Prerequisites
+
+* An Azure Communication Resource, learn how to create one from [Create an Azure Communication Resource](https://docs.microsoft.com/azure/communication-services/quickstarts/create-communication-resource)
+  higher to use this library.
```
Comment (angiurgiu) [inline sdk/communication/azure-communication-chat/README.md:10]: Something seems to be missing here, copy paste error?

"  higher to use this library"

File: sdk/communication/azure-communication-chat/README.md [hunk 2]
```diff
@@ -0,0 +1,712 @@
...
+
+```xml
+<dependency>
+  <groupId>com.azure.android</groupId>
+  <artifactId>azure-communication-chat</artifactId>
+  <version>1.0.0-beta.1</version>
+</dependency>
+```
+
+## User and User Access Tokens
```
Comment (angiurgiu) [inline sdk/communication/azure-communication-chat/README.md:58]: nit: "User**s** and User Access Tokens"

File: sdk/communication/azure-communication-chat/README.md [hunk 3]
```diff
@@ -0,0 +1,712 @@
...
+## Prerequisites
+
+* An Azure Communication Resource, learn how to create one from [Create an Azure Communication Resource](https://docs.microsoft.com/azure/communication-services/quickstarts/create-communication-resource)
+  higher to use this library.
+* The library is written in Java 8. Your application must be built with Android Gradle Plugin 3.0.0 or later, and must
+  be configured to
+  [enable Java 8 language desugaring](https://developer.android.com/studio/write/java8-support.html#supported_features)
+  to use this library. Java 8 language features that require a target API level >21 are not used, nor are any Java 8+
+  APIs that would require the Java 8+ API desugaring provided by Android Gradle plugin 4.0.0.
+* You must have an [Azure subscription](https://azure.microsoft.com/free/) to use this library.
```
Comment (angiurgiu) [inline sdk/communication/azure-communication-chat/README.md:16]: Could also mention creating an Azure Resource, some of the other Readmes do

File: sdk/communication/azure-communication-chat/README.md [hunk 4]
```diff
@@ -0,0 +1,712 @@
...
+```
+
+## User and User Access Tokens
+
+User access tokens enable you to build client applications that directly authenticate to Azure Communication Services. Refer [here](https://docs.microsoft.com/azure/communication-services/quickstarts/access-tokens) to learn how to create a user and issue a User Access Token.
+
+The id for the user created above will be necessary later to add said user as a member of a new chat thread. The initiator of the create request must be in the list of members of the chat thread.
+
+## Create the AzureCommunicationChatClient
+
```
Comment (angiurgiu) [inline sdk/communication/azure-communication-chat/README.md:65]: This is probably fine for now, but we should use snippets from real code using the embedding tool

File: sdk/communication/azure-communication-chat/README.md [hunk 5]
```diff
@@ -0,0 +1,712 @@
...
+        }
+    });
+}
+```
+
+# Troubleshooting
+
+## General
+
+The client raises via Callback.onFailure.
```
Comment (angiurgiu) [inline sdk/communication/azure-communication-chat/README.md:691]: raises Exceptions?

File: sdk/communication/azure-communication-chat/README.md (hunk 3 above)
Comment (anuchandy) [reply sdk/communication/azure-communication-chat/README.md:16]: its there as the first point, I'm reordering the bullet points

File: sdk/communication/azure-communication-chat/README.md (hunk 1 above)
Comment (anuchandy) [reply sdk/communication/azure-communication-chat/README.md:10]: yep, copy-paste error fixing it

**Pull request merged**